src/server/scripts/World/mob_generic_creature.cpp

```diff
@@ -27,7 +27,10 @@ EndScriptData */
 #include "ScriptedCreature.h"
 #include "PassiveAI.h"
 
-#define GENERIC_CREATURE_COOLDOWN   5000
+enum GenericCreature
+{
+    GENERIC_CREATURE_COOLDOWN = 5000
```

Comment:
and I would also need to
```cpp
#include <chrono>
```
right? BTW, there are only 3 occurrences where it is used in this script.
It would be just as clean to insert `std::chrono::seconds(5)` 3 times.